internal/helm/repository/chart_repository.go

```
@@ -281,9 +285,7 @@ func (r *ChartRepository) DownloadChart(chart *repo.ChartVersion) (*bytes.Buffer
 
 	t := transport.NewOrIdle(r.tlsConfig)
 	clientOpts := append(r.Options, getter.WithTransport(t))
-	defer func() {
-		_ = transport.Release(t)
-	}()
+	defer transport.Release(t)
```

golangci-lint (2.13.2)

[error] Error return value of `transport.Release` is not checked

(errcheck)